src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -994,89 +996,94 @@ private void UpdateMediaControlState(DependencyProperty property)
 					break;
 
 				case var _ when property == IsRepeatButtonVisibleProperty:
-					BindVisibility(m_tpRepeatButton, IsRepeatButtonVisible);
+					BindVisibility(m_tpRepeatButton, IsImplemented(typeof(MediaPlayer), "IsLoopingEnabled") && IsRepeatButtonVisible, property);
```

Comment:
So all this does is force a false value when something is not implemented?

I think a better idea than `IsImplemented` is to move the property out from the generated file, and keep the NotImplementedAttribute, but return false instead of throwing in the getter. What do you think?

Reply:
I started to use the ApiInformation.IsPropertyPresent.
Jerome show to me.